### PR TITLE
retain errors from the response

### PR DESCRIPTION
**Pull request by latompa**

### CODE REVIEW

File: lib/eligible.rb [hunk 1]
```diff
@@ -237,10 +237,11 @@ def self.error_message(error)
 
   def self.handle_api_error(rcode, rbody)
     begin
-      error_obj = Eligible::JSON.load(rbody)
-      error_obj = Util.symbolize_names(error_obj)
-      fail EligibleError unless error_obj.key?(:error)
+      error_obj = Util.symbolize_names(Eligible::JSON.load(rbody))
+      fail EligibleError unless error_obj.keys.any?{|k| %i(error errors).include? k}
```
Comment (kovyrin) [inline lib/eligible.rb]: I'd recommend explaining here why we fail. Maybe something like `fail EligibleError, "Could not find an error key in the response"` or something like that?

File: lib/eligible.rb [hunk 2]
```diff
@@ -249,24 +250,24 @@ def self.handle_api_error(rcode, rbody)
 
     case rcode
     when 400, 404 then
-      raise invalid_request_error(error_msg, rcode, rbody, error_obj)
+      raise invalid_request_error(error_msg, rcode, rbody, error_obj, errors)
```
Comment (kovyrin) [inline lib/eligible.rb]: if we're passing the `error_obj` here, it looks like errors could be loaded from it as needed, why pass it separately?

**Comment by hari-eligible** [comment]
LGTM, we need update README by documenting multiple errors

**Comment by ankitdimania-eligible** [comment]
closing in favour of https://github.com/eligible/eligible-ruby/pull/43

**Pull request closed without merging**